packages/blade/src/components/Dropdown/docs/DropdownWithButton.stories.tsx

```diff
@@ -179,7 +179,7 @@ export const InternalAutoPositioning = (): React.ReactElement => {
         <Dropdown>
           <DropdownButton>Top Right Menu</DropdownButton>
           <DropdownOverlay>
-            <ActionList surfaceLevel={3}>
```

Comment:
So now that we've removed surfaceLevel prop, are we introducing color prop?

How will existing consumers use different surface levels?

Reply:
Also, breaking change. We would need a codemod

Reply:
> How will existing consumers use different surface levels?

Fun fact: In design we removed the surfaceLevel option earlier only. So all designs were always surfaceLevel={2} and this prop was not needed. 

Even if someone is using surfaceLevel={3}, its probably not following design so should be fine to change it to static color. Anyway visually also its just color change and doesn't look broken if someone was using it.

> Also, breaking change. We would need a codemod

Yup we have added it in our codemod already